paparazzi/api/paparazzi.api

```diff
@@ -213,9 +213,9 @@ public final class app/cash/paparazzi/SnapshotHandlerKt {
 
 public final class app/cash/paparazzi/SnapshotVerifier : app/cash/paparazzi/SnapshotHandler {
 	public static final field $stable I
-	public fun <init> (D)V
-	public fun <init> (DLjava/io/File;)V
-	public synthetic fun <init> (DLjava/io/File;ILkotlin/jvm/internal/DefaultConstructorMarker;)V
+	public fun <init> (DLapp/cash/paparazzi/internal/Differ;)V
```

Comment:
This puts an internal type into public API. The type should probably be moved somewhere public.

Reply:
addressed in https://github.com/cashapp/paparazzi/pull/2009